packages/core/__tests__/client.ts

```diff
@@ -58,6 +61,18 @@ describe('Sentry.Client', () => {
     expect(spy2).toHaveBeenCalledTimes(1);
   });
 
+  test('multiple install calls on Adapter should only call once', async () => {
+    expect.assertions(2);
+    const sentry = new Sentry.Client(dsn);
+    sentry.use(MockAdapter, { testOption: true });
+    const spy1 = jest.spyOn(sentry, 'install');
+    const spy2 = jest.spyOn(sentry.getAdapter(), 'install');
+    await sentry.install();
+    await sentry.install();
+    expect(spy1).toHaveBeenCalledTimes(2);
```

Comment:
This assertion looks redundant, because you trigger those calls yourself 2 lines above